db/seeds/source_types.rb

```diff
@@ -99,6 +99,9 @@ def update_or_create(attributes)
     :name   => "Tenant ID, Client ID, Client Secret",
     :fields => [
       {:component => "text-field", :name => "authentication.authtype", :hideField => true, :initializeOnMount => true, :initialValue => "tenant_id_client_id_client_secret"},
+      {:component => "text-field", :name => "billing_source.data_source.storage_account", :label => "Storage account name"},
```

Comment:
Please, provide here and below some `stepKey`, I understand that you cannot provide the correct `stepKey` because fields are appended to the end of the step, but you should provide at least some key to not showing this fields in other applications. You can do something like `:stepKey => 'cost-management-specific'`

Reply:
You're right. I forgot to add them there. Thanks!